tests/lib/pkgdb.sh

```diff
@@ -520,6 +520,12 @@ distro_install_build_snapd(){
             fi
         fi
 
+        if [[ "$SPREAD_SYSTEM" == opensuse-tumbleweed-* ]]; then
+            # Package installation applies vendor presets only, which leaves
+            # snapd.apparmor disabled.
+            systemctl enable --now snapd.apparmor.service
```

Comment:
Should we fix that in the package somehow?

Reply:
We have a relevant line calling `%{service_add_post}`. Nonetheless I spotted another bug in packaging in the %post section.